docs/changelog.rst

```diff
@@ -15,6 +15,8 @@ New flavors:
 
 New fields for existing flavors:
 
+- Added ARCBUField model field.
```

Comment:
You actually added a form field, not a model field. :) Can you change this? Thanks.